packages/spindle-ui/src/Pagination/Pagination.css

```
@@ -0,0 +1,113 @@
+/*
+ * Pagination
+*/
+.spui-Pagination {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
+
+.spui-Pagination-list {
+  align-items: center;
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.spui-Pagination-item {
+  align-items: center;
+  display: flex;
+  margin: 0 5px;
+}
+
+.spui-Pagination-button {
+  align-items: center;
+  background: none;
+  border: none;
+  border-radius: 12px;
+  box-sizing: border-box;
+  display: flex;
+  font-size: 0.875em;
+  height: 40px;
+  justify-content: center;
+  line-height: 1.3;
+  margin: 0;
+  padding: 0;
+  -webkit-tap-highlight-color: var(--Button-tapHighlightColor);
+  width: 40px;
+}
+
+.spui-Pagination-button:disabled {
+  opacity: 0.3;
+}
+
+.spui-Pagination-button:not([aria-current]) {
+  background: var(--color-surface-tertiary);
+  color: var(--color-text-medium-emphasis);
+  font-weight: bold;
+}
+
+.spui-Pagination-button:focus:not(:focus-visible) {
+  outline: none;
+}
+
+.spui-Pagination-button--hasLabel {
+  width: auto;
+}
+
+.spui-Pagination-item .spui-Pagination-button[aria-current] {
+  background: var(--color-surface-primary);
+  border: solid 1px var(--color-border-low-emphasis);
+  color: var(--color-text-low-emphasis);
+}
+
+.spui-Pagination-item .spui-Pagination-button:focus {
+  outline: 2px solid var(--color-focus-clarity);
+  outline-offset: 1px;
+}
+
+@media (hover: hover) {
+  .spui-Pagination-item .spui-Pagination-button:not([aria-current]):hover {
+    background-color: var(--gray-20-alpha);
```

```suggestion
    background-color: var(--color-surface-quaternary);
```
`--color-surface-quaternary` にしても良さそうですかね〜。